kiva/fonttools/font_manager.py

```diff
@@ -523,7 +524,7 @@ def ttfFontProperty(fpath, font):
     *font* is a :class:`FT2Font` instance.
     """
     props = getPropDict(font)
-    name = props[(1, 0, 0, 1)]
+    name = props[(1, 0, 0, 1)].decode()
```

Comment:
In python 2, `decode()` is a no-op if the value is already a unicode, in python 3 it raises an AttributeError.

The getPropDict will generate a "dictionary" of bytes, that needs to be decoded at this point. This module is quite sketchy in terms of unicode vs bytes. It should probably be refactored in some ways. This is the best I could come up with minimal code change.